src/vs/editor/contrib/inlineCompletions/browser/ghostTextWidget.ts

```diff
@@ -202,7 +203,8 @@ class AdditionalLinesWidget extends Disposable {
 	constructor(
 		private readonly editor: ICodeEditor,
 		private readonly languageIdCodec: ILanguageIdCodec,
-		private readonly lines: IObservable<{ targetTextModel: ITextModel; lineNumber: number; additionalLines: LineData[]; minReservedLineCount: number } | undefined>
+		private readonly lines: IObservable<{ targetTextModel: ITextModel; lineNumber: number; additionalLines: LineData[]; minReservedLineCount: number } | undefined>,
+		private readonly cls: string
```

Comment:
Can you use a more descriptive value for this string and/or add docs?
Maybe just `className`, with docs being "the css class of the view-lines"?
Also, the value should be validated to prevent escaping issues (see DecorationOptions for validation examples).

Reply:
Oh, I've realized we don't need this at all - it's leftover from one of our experiments where we were trying to do background colouring also for ghost text. This turned out to be too heavy/distracting. 

I removed this change to minimize diff.